src/conversation_engine/engine.py

```diff
@@ -40,23 +44,23 @@ def sanitize_text(txt):
 
 def describe_current_subtask(subtask, prefix=True):
     """Make a 'natural' language description of subtask name"""
-    to_verb = { "AnswerQuestion": "answering a question",
-                "ArmGoal": "moving my arm",
-                "DemoPresentation": "giving a demo",
-                "Find": "finding",
-                "Follow": "following",
-                "Guide": "guiding",
-                "GripperGoal": "moving my gripper",
-                "HandOver": "handing something over",
-                "Inspect": "inspecting",
-                "LookAt": "looking",
-                "NavigateTo": "navigating",
-                "PickUp": "picking up",
-                "Place": "placing",
-                "ResetWM": "resetting my world model",
-                "Say": "speaking",
-                "SendPicture": "sending a picture",
-                "TurnTowardSound": "turning towards a sound"}
+    to_verb = {"AnswerQuestion": "answering a question",
+               "ArmGoal": "moving my arm",
+               "DemoPresentation": "giving a demo",
+               "Find": "finding",
+               "Follow": "following",
+               "Guide": "guiding",
+               "GripperGoal": "moving my gripper",
+               "HandOver": "handing something over",
+               "Inspect": "inspecting",
+               "LookAt": "looking",
+               "NavigateTo": "navigating",
+               "PickUp": "picking up",
+               "Place": "placing",
+               "ResetWM": "resetting my world model",
+               "Say": "speaking",
+               "SendPicture": "sending a picture",
+               "TurnTowardSound": "turning towards a sound"}
```

Comment:
Looks like something that should be in the actions themselves as they provide the link between semantic action and language. I guess this is OK for now...

Reply:
Doing this properly is https://github.com/tue-robotics/conversation_engine/issues/11